src/GovUk.Education.ExploreEducationStatistics.Content.Api/appsettings.Development.json

```diff
@@ -1,6 +1,7 @@
 {
   "ConnectionStrings": {
-    "PublicStorage": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://data-storage:10000/devstoreaccount1;QueueEndpoint=http://data-storage:10001/devstoreaccount1;"
+    "PublicStorage": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://data-storage:10000/devstoreaccount1;QueueEndpoint=http://data-storage:10001/devstoreaccount1;",
+    "PublisherStorage": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;QueueEndpoint=http://data-storage:10001/devstoreaccount1;"
```

Comment:
Does this need adding to appsettings.json as well, or do we have to somehow otherwise prepare environments that this code will go out to ahead of time so they have these connection details?

Reply:
It's not used in any of the environments. Locally, on startup, a message is queued to regenerate the content cache so that we always have the latest content. Sending this message needs the storage connection.

It's wrapped in `env.IsDevelopment()` which returns false in all of our environments.